### ISSUE TITLE
Update podcasts without visiting the page.

### ISSUE THREAD

**Issue by Ethanol6** [SamTV12345/PodFetch#603]
### Clear and concise description of the problem

The automatic podcast update option says it updates automatically when the page is visited. This means that podcast apps that use PodFetch don't get updated podcasts unless you open the page.

My use case is such that I never open the podcast page. Even if I did, I would need to open 25+ pages just so AntennaPod on my phone can download new episodes before I head to work. 

Manually navigating to the page just so it can automatically update seems counter-intuitive. 

### Suggested solution

I think it would make sense if automatic update was truly automatic, requiring no navigation to the pages at all. If you choose to manually update podcasts there should be a global update button somewhere easy to access that would update all podcasts with a single click. Another option could be that podcasts do not update automatically unless the global update button is pressed OR you navigate to the page in which case each podcast is updated only as you visit the page.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/SamTV12345/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/SamTV12345/.github/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by SamTV12345** [comment]
> ### Clear and concise description of the problem
> The automatic podcast update option says it updates automatically when the page is visited. This means that podcast apps that use PodFetch don't get updated podcasts unless you open the page.
> 
> My use case is such that I never open the podcast page. Even if I did, I would need to open 25+ pages just so AntennaPod on my phone can download new episodes before I head to work.
> 
> Manually navigating to the page just so it can automatically update seems counter-intuitive.
> 
> ### Suggested solution
> I think it would make sense if automatic update was truly automatic, requiring no navigation to the pages at all. If you choose to manually update podcasts there should be a global update button somewhere easy to access that would update all podcasts with a single click. Another option could be that podcasts do not update automatically unless the global update button is pressed OR you navigate to the page in which case each podcast is updated only as you visit the page.
> 
> ### Alternative
> _No response_
> 
> ### Additional context
> _No response_
> 
> ### Validations
> * [x]  Follow our [Code of Conduct](https://github.com/SamTV12345/.github/blob/main/CODE_OF_CONDUCT.md)
> * [x]  Read the [Contributing Guide](https://github.com/SamTV12345/.github/blob/main/CONTRIBUTING.md).
> * [x]  Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

I'm not sure. Could be that the option is misleading. The backend fetches in your specified time interval new episodes.

**Comment by Ethanol6** [comment]
When I visit the Podfetch home page the "latest episodes" list show episodes that are days, weeks, months old. There are no episodes listed that are newer than the last time I visited the podcast page in Podfetch. I also have automatic update turned on. The option is supposed to "PodFetch will automatically check for new episodes when the page is visited." However, upon visiting a specific podcast page or the "All Subscriptions" page the podcast(s) doesn't update after waiting several minutes and refreshing the page. I must always click the update icon next to the podcast title (or update them all from the "All Subsctiptions" page) and refresh the webpage in order to update the podcast.

The polling interval is set to 60 in my docker-compose file but nothing seems to suggest it actually is refreshing every 60 minutes. Using Antennapod to fetch episodes from the Podfetch feed only has episodes that also appear in the "latest episodes" list and those can be weeks old.

**Comment by SamTV12345** [comment]
Has this somehow fixed itself? I've got PodFetch deployed on my Raspberry PI. Unfortunately I didn't have time to listen to a podcast for a while but it still shows me all podcasts with their latest update.

**Comment by Ethanol6** [comment]
Not exactly. On the Home screen I do see the most recent episodes of only one podcast. That podcast is whichever one Podfetch calls number 1. 

On the All Subscriptions page the URL for that podcast is `server.com/ui/podcasts/1/episodes`

The other podcasts were last updated months ago, only showing up on the Home page when manually refreshed.

**Comment by Ethanol6** [comment]
Interesting development. My podcasts were all imported from an OPML file exported from PodGrab. I deleted all my podcasts and data and manually re-added them using the Podfetch interface (using the iTunes database) and it seems like the podcasts are staying up to date now.

**Comment by Patriot2407** [comment]
> Interesting development. My podcasts were all imported from an OPML file exported from PodGrab. I deleted all my podcasts and data and manually re-added them using the Podfetch interface (using the iTunes database) and it seems like the podcasts are staying up to date now.

I also had better luck with this as well, have had to do this for several of mine. I have only 1 that is being troublesome, but I think its unique to this podcast

**Comment by Patriot2407** [comment]
My podcasts do not update automatically either, unless I click the refresh button on the all subscriptions page

**Comment by Patriot2407** [comment]
Podcasts still do not auto-update. Is there something that needs to be done? I have even exported out the original link OPML file from podfetch, completely built a new podfetch instance and imported that OPML, and I am still not seeing podcasts auto update unless I manually cick update all.

**Comment by SamTV12345** [comment]
You just have to have it checked in the settings page or in the podcast settings menu.

**Comment by Patriot2407** [comment]
Yes, its working
